docs/CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * `amethyst_renderer::Rgba` is now a `Component` that changes the color and transparency of the entity
 it is attached to. ([#1282])
 * `AutoFov` and `AutoFovSystem` to adjust horizontal FOV to screen aspect ratio. ([#1281])
-
+* Add `icon` to `DisplayConfig` to set a window icon using a path to a file ([#1373])
```

Comment:
Hey you also need to add a line for this link down near line 56.

Reply:
Thanks